plugins/action_update_review_status/plugin.json

```diff
@@ -4,6 +4,7 @@
   "name": "Update Review Statuses",
```

Comment:
Out of sync with other files related - there is one folder `action_update_review_status`. The description of this action (on `List-of-Action-plugins.md`)  should reflect the name of this folder properly.

Reply:
I'm not sure what you mean -- the name matches the Section heading in the doc file: "### Update Review Statuses" -- line 204 of `List-of-Action-plugins.md`.

Is that what you mean?

Reply:
Yes, I was mentioning the action name actually, it's name `updateReviews` where it should be really `updateReviewsStatuses`. But not very important. We can leave it as it is.

Reply:
Ah, righto. Easily done ccd2bf17